docs/user/merging-feeds.md

```diff
@@ -25,37 +25,42 @@ There are a set of rules that govern the requirements for input feed versions an
   one is automatically generated for the merged feed.
 1. Stops will be merged on `stop_code` or `stop_id` if `stop_code` is missing. However, some restrictions apply on
   when missing stop_code values are permitted:
-    1. Stops with `location_type` greater than `0` (i.e., anything but `0` or `empty`) are permitted
-       to have empty `stop_codes` (even if there are other stops in the feed that have
-       `stop_code` values). This is because these location_types represent special entries
-       that are either stations, entrances/exits, or generic nodes (e.g., for
-       `pathways.txt`). The merge will happen on `stop_code` if provided, or fallback on stop_id.
-    2. For regular stops (`location_type = 0` or empty), all or none of the stops must
-       contain `stop_codes`. Otherwise, the merge feeds job will be failed.
+
+    (i) Stops with `location_type` greater than `0` (i.e., anything but `0` or `empty`) are permitted
+    to have empty `stop_codes` (even if there are other stops in the feed that have
+    `stop_code` values). This is because these location_types represent special entries
+    that are either stations, entrances/exits, or generic nodes (e.g., for
+    `pathways.txt`). The merge will happen on `stop_code` if provided, or fallback on stop_id.
```

Comment:
add code block for `stop_id`

Reply:
Thanks, good catch, fixed in https://github.com/ibi-group/datatools-ui/pull/757/commits/caec4a1099e14344383610120a94992eb4b1191d.